README.md

```diff
@@ -3,3 +3,13 @@
 * You'll run a dedicated browser in another window that's controlling ChatGPT.
 * Two terminals: `go run main.go`, and `python server.py`. I am extremely doubtful they will work for you on the first run.
 * This marks the end of the readme file; it is a bit sparse; thankfully the code is too! Just tuck in if you can... and I will try to add more here later.
+
+## Get Started
+
+1. Install Requirements (suggest to do it in `virtualenv` or `conda`)
```

Comment:
how about just sticking to `venv` and may be write instructions for the same? I feel instead of presenting many options, just present one option with all the instructions, so it's easy for people who are out of py ecosystem

Reply:
Fair - although, I personally use Conda so I didn't want to write something that I didn't test personally.

Reply:
Fair - although I personally use Conda, so I didn't want to write up something I didn't use / test personally.

Happy to remove that line if you think it's confusing.